msi/GitProduct.wxs

```diff
@@ -290,9 +292,12 @@
       <Custom Action="ExecConfigureSslCAInfo" Before="ExecConfigureCoreAutoCrlf">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
-      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecPostInstallBat">
+      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecConfigureFsCache">
```

Comment:
Stop

Sent from my iPhone

> On May 9, 2016, at 9:27 AM, Johannes Schindelin <notifications@github.com> wrote:
> 
> In msi/GitProduct.wxs:
> 
> > @@ -290,9 +292,12 @@
> >        <Custom Action="ExecConfigureSslCAInfo" Before="ExecConfigureCoreAutoCrlf">
> >          <![CDATA[REMOVE <> "ALL"]]>
> >        </Custom>
> > -      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecPostInstallBat">
> > +      <Custom Action="ExecConfigureCoreAutoCrlf" Before="ExecConfigureFsCache">
> Since the order of these custom actions does not really matter, why not keep the order in which they were added? Or even the reverse order, so that existing custom actions do not need to flip-flop their attributes when other custom actions are added.
> 
> Also, I have to admit that I am more and more in favor of adding the UI only when the corresponding custom action is introduced. That would make for a much easier read (or for that matter, review).
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>